rockcraft/project.py

```diff
@@ -498,5 +498,10 @@ def _add_pebble_data(yaml_data: Dict[str, Any]) -> None:
         "plugin": "go",
         "source": "https://github.com/canonical/pebble.git",
         "build-snaps": ["go/1.19/stable"],
+        "override-build": (
+            "go mod download\n"
+            'CGO_ENABLED=0 go build -ldflags="-w -s" -o pebble ./cmd/pebble\n'
+            'install -D -m755 pebble "$CRAFT_PART_INSTALL"/bin/pebble\n'
```

Comment:
should we put it in `/usr/bin/` instead? At least while a solution for preserving symlinks is being worked on...

Reply:
sure, updated

Reply:
Putting in /usr/bin is interesting, as if the base is bare it is a strange directory structure to have. After some thought, I believe we need a rocks owned directory (we already do).

This is good enough for now, but let's make sure we talk about this in one of our live conversations